test/cypress/e2e/register_challenge.spec.cy.js

```diff
@@ -0,0 +1,298 @@
+describe('Register Challenge page', () => {
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.visit('/#/register-challenge');
```

Comment:
Use named route here, not hardcoded URL string path, please.

Reply:
Solution:

1. Create ./src/router/routes_conf.ts file

This is example config for register-challenge router path, you must extend `routesConf` object with another router paths e.g. home and etc.

```ts
const routesConf: object = {
    'register-challenge': {
       path: '/register-challenge',
       children: {
         name: 'register-challenge',
       },
    },
};

export { routesConf };
```

2. Adjust ./src/router/routes.ts file

```ts
import { routesConf } from './routes_conf';

...

  {
    path: routesConf['register-challenge']['path'],
    component: () => import('layouts/LoginRegisterLayout.vue'),
    children: [
      {
        path: '',
        component: () => import('pages/RegisterChallengePage.vue'),
        name: routesConf['register-challenge']['children']['name'],
      },
    ],
  },
```

3. Adjust e2e test file ./test/cypress/e2e/register_challenge.spec.cy.js

```js
import { routesConf } from '../../../src/router/routes_conf';

describe('Register Challenge page', () => {
  context('desktop', () => {
    beforeEach(() => {
      cy.visit(routesConf['register-challenge']['path']);
      cy.viewport('macbook-16');
    });
```

Reply:
Updated.
We still need to prepend `'#'` to navigate to the correct route, but the route is dynamic.
I will update this for other E2E test in a separate PR.